changelog.md

```diff
@@ -1,13 +1,34 @@
 # Changelog
 
-## Unreleased
+## SFML 2.6.2
```

Comment:
SFML 2.6.2 is still unreleased 😄

Reply:
I figured I would omit the "Unreleased" so that we have one fewer thing to change when we do release. What are the odds we forget to remove that word when we ship?